### ISSUE TITLE
Accent keys issue

### ISSUE THREAD

**Issue by jouven** [TurboVNC/turbovnc#307]
Context:
Client and Host are on Turbovnc 2.2.7, both on Debian unstable, both use Xfce, host has also VirtualGL 3.0 installed
To connect I execute /opt/TurboVNC/bin/vncviewer (checking the about window it's the java viewer)

The issue I have is that accent characters don't work (I hit the keys but nothing gets written), with or without letters, e.g., à é or just ` ´
Shift modified of the accent keys don't work either, with or without letters, ä ê or ¨ ^
AltGr modified does work [ {
The rest works 1:1 (Spanish keyboard), ñ, ç, €, all the AltGr + number combinations
If I use plain terminal SSH everything works

```
localectl status
```
returns:
```
   System Locale: LANG=en_GB.UTF-8
                  LANGUAGE=en_GB:en
       VC Keymap: n/a
      X11 Layout: es
       X11 Model: pc105
```
For client and host (executing the command from Turbovnc or just plain SSH)

**Comment by dcommander** [comment]
Duplicate of https://github.com/TurboVNC/turbovnc/issues/194.  In my opinion, the most expedient way to resolve these issues with dead keys/accent keys is to complete https://github.com/TurboVNC/turbovnc/issues/108, but I need funding for that.

**Comment by dcommander** [comment]
Worked around via the implementation of #108, which is now available in the dev/3.1 evolving [pre-release build](https://turbovnc.org/DeveloperInfo/PreReleases).  Please test thoroughly.  (You need to use both the server and viewer from the pre-release build in order to activate the feature.)
